plugins/node.d.freebsd/cpu

```diff
@@ -34,9 +34,10 @@ if [ "$1" = "autoconf" ]; then
 fi
 
 if [ "$1" = "config" ]; then
-
 	OSV=`$SYSCTL_BIN -n kern.osrelease | cut -f1 -d.`
-	if [ "$OSV" = "4" ]; then
+	if [ $(uname -s) = "MidnightBSD" ]; then
+	  STATUNITS=`$SYSCTL_BIN -n kern.clockrate | cut -f13 -d' '`
```

Comment:
I think, you mixed up tabs and spaces here.

Additionally it would be great, if you could quote `$(uname -s)` in order to avoid and potential whitespace issues (and to please `shellcheck`).

Reply:
After taking another look at it: maybe you can just add the condition to the branch for `"$OSV -ge "5"` below?

Reply:
I changed the order so that the  $(uname -s) = "MidnightBSD" || "$OSV -ge "5" check is before "$OSV" = "4" instead. Otherwise, when MidnightBSD hits version 4, things will break.